Controller/Payment/Webhook.php

```diff
@@ -198,12 +199,12 @@ protected function authorize(array $post)
         }
         catch (\Razorpay\Api\Errors\Error $e)
         {
-            $this->logger->critical("Razorpay Webhook: fetching RZP order data(id:$rzpOrderId) "
```

Comment:
@enishant , this should be critical only.

Reply:
@ramth05 updated exception logs to critical